modules/RTC/TraceablePeerConnection.js

```diff
@@ -355,9 +355,8 @@ export default function TraceablePeerConnection(
 
     if (this.maxstats) {
         this.statsinterval = window.setInterval(() => {
-            this.getStats(stats => {
-                if (stats.result
-                    && typeof stats.result === 'function') {
+            this.getStats().then(stats => {
+                if (typeof stats?.result === 'function') {
```

Comment:
Question: when is this the case? I thought this was with ancient stats?

Reply:
Nope, these are being used by torture testcases.